### DISCUSSION TITLE
ALTUS always crashes when I open a video and play it

### DISCUSSION THREAD

**Discussion by Ragnaroeko** [amanharwara/altus#177] [Q&A]
Since the last 2 or 3 updates, ALTUS always crashes when I open a video and play it in Whats app, even the latest version 4.5.3. Before that I had no problems. I have Windows 7 with all the updates. Greetings from Germany

**Comment by Ragnaroeko** [comment]
Problemsignatur:
  Problemereignisname:	APPCRASH
  Anwendungsname:	Altus.exe
  Anwendungsversion:	4.5.3.0
  Anwendungszeitstempel:	6192fabb
  Fehlermodulname:	Altus.exe
  Fehlermodulversion:	4.5.3.0
  Fehlermodulzeitstempel:	6192fabb
  Ausnahmecode:	c0000005
  Ausnahmeoffset:	00000000004e09e3
  Betriebsystemversion:	6.1.7601.2.1.0.256.1
  Gebietsschema-ID:	1031
  Zusatzinformation 1:	077c
  Zusatzinformation 2:	077c6f548d4a3a00f914bd1e495cc696
  Zusatzinformation 3:	c8ad
  Zusatzinformation 4:	c8ad5e2d983985ee6d8ef146c11a6d28

**Comment by amanharwara** [comment]
(If there is an issue with Altus, please open an actual issue whenever possible instead of starting a discussion.)

Btw have you tried downgrading to an older version and seeing which one works?

**Comment by Ragnaroeko** [comment]
I've tried around now:

Altus 4.5.1 is working! on Windows 7 64bit current version, Browser Google Chrome!

From/away Altus version 4.5.2, Altus crashes when playing videos, (app crash!)
Also the latest version of Altus 4.5.4 crash when playing video.

So there must have been a change / error from Altus 4.5.2 that caused this. Unfortunately I don't know how to tell the author of this. Greetings to everyone from Germany
